src/Aspire.Hosting/Dashboard/DashboardExtensions.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting.Dashboard;
+
+/// <summary>
+/// Provides extension methods for adding the Aspire dashboard to an <see cref="IDistributedApplicationBuilder"/>.
+/// </summary>
+public static class DashboardExtensions
+{
+    public static IResourceBuilder<ExecutableResource> AddDashboard(this IDistributedApplicationBuilder builder)
```

Comment:
Should this be in P3?

Reply:
Removed. It was WIP code that stuck around. Thanks.